src/Relation/Nullary/Decidable.agda

```diff
@@ -115,3 +116,20 @@ module _ {p} {P : Set p} where
   from-no : (p : Dec P) → From-no p
   from-no (no ¬p) = ¬p
   from-no (yes _) = _
+
+------------------------------------------------------------------------
+-- Result of decidability
+
+module _ {P : Set p} where
+
```

Comment:
Can you not drop the anonymous module? `P` is part of the variable block already!